server/models/models.games.js

```diff
@@ -65,10 +65,24 @@ const insertCommentByReviewId = (id, comment) => {
     })
 }
 
+const updateReviewVotesById = (id, inc) => {
```

Comment:
You may need some error handling in the model for invalid id and/or missing/invalid keys on the input body